.pre-commit-config.yaml

```diff
@@ -237,3 +231,10 @@ repos:
         additional_dependencies:
         - flake8==4.0.1
         - flake8-pyi==22.5.1
+    -   id: future_annotations
+        name: import annotations from __future__
+        entry: 'from __future__ import annotations'
+        language: pygrep
+        args: [--negate]
```

Comment:
`negate` checks that the pattern in `entry` is always present (as opposed to checking it's absent)